Cargo.toml

```diff
@@ -1,12 +1,14 @@
 [package]
-name = "linked_list"
-version = "0.1.0"
+name = "linked_list_r4l"
+version = "0.2.0"
 edition = "2021"
-authors = ["Wedson Almeida Filho <wedsonaf@google.com>"]
+authors = ["Wedson Almeida Filho <wedsonaf@google.com>", "WeiKang Guo <guoweikang.kernel@gmail.com>"]
 description = "Linked lists that supports arbitrary removal in constant time"
 license = "GPL-2.0-or-later"
 homepage = "https://github.com/arceos-org/arceos"
 repository = "https://github.com/arceos-org/linked_list"
```

Comment:
Rename this repo to `linked_list_r4l`

Reply:
Ok, but renaming the repository requires you to complete this operation